config/peerpods/peerpodssecret.yaml

```diff
@@ -9,3 +9,4 @@ stringData:
   #LIBVIRT_URI: "qemu+ssh://root@192.168.122.1/system?no_verify=1"
   #LIBVIRT_NET: "default
   #LIBVIRT_POOL: "default"
+  #REDHAT_OFFLINE_TOKEN: "" #Required to download rhel base image
```

Comment:
maybe mention also how to get it (https://access.redhat.com/management/api IIUC)

Reply:
Will it support getting the image from container payload? if not, consider validating REDHAT_OFFLINE_TOKEN is set (IIUC it will be mandatory)